client/src/components/layout/Question.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import { Typography } from '@material-ui/core';
+
+import ReactMarkdown from 'react-markdown';
+import {Prism as SyntaxHighlighter} from 'react-syntax-highlighter';
+import gfm from 'remark-gfm';
+
+
+const components = {
+  code({node, inline, className, children, ...props}) {
+    const match = /language-(\w+)/.exec(className || '')
```

Comment:
Can you run Prettier on your files, should fix all the code styling issues and missing semicolons

Reply:
Sure, I will see what is going on with the Prettier extension as it is not formatting my files properly.